modules/mobile/spec/support/schemas/VAOS_v2_appointments.json

```diff
@@ -230,7 +230,8 @@
     },
     "meta": {
       "required": [
```

Comment:
This schema is not entirely accurate but I'm not fixing it as part of this ticket.